SignalUI/Appearance/Theme+OWS.swift

```diff
@@ -293,6 +295,11 @@ public extension Theme {
             return isDarkThemeEnabled ? "camera-solid-24" : "camera-outline-24"
         case .micButton:
             return isDarkThemeEnabled ? "mic-solid-24" : "mic-outline-24"
+        case .speakerButton:
+            return "speaker-solid-28" // TODO: Add outline for dark theme or select another asset.
+        case .stopButton:
+            return "pause-filled-24" // TODO: Use correct asset
```

Comment:
@EvanHahn-Signal I just remembered that we did not have a conversation about this. Will you be plugging in the correct assets for this prior to merging to your internal repo?

Reply:
Yup, that's all sorted! Thanks for checking.

Reply:
Thanks for the update. For future contributions, is leaving `TODO` comments in the code the best way to flag stuff like this or is there a more preferable way?